### PR TITLE
Add Heartbeat

### PR DESCRIPTION
**Pull request by syphernl**
This PR adds the Heartbeat (shipper for uptime monitoring).

https://www.elastic.co/products/beats/heartbeat

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 beats-formula
 =============
 
+0.1.2 (2019-04-05)
```
Comment (psmiraglia) [inline CHANGELOG.rst]: You should update also the `VERSION` file with value `0.1.2`.

File: pillar.example [hunk 2]
```diff
@@ -91,6 +91,15 @@ beats:
             output.logstash:
               hosts: ["localhost:9200"]
             logging.to_files: false
+    heartbeat:
+        config:
+            heartbeat.monitors:
+                - type: http
+                  urls: ["http://localhost:9200"]
+                    schedule: "@every 10s"
```
[the remark this answers is not in the document]
Comment (syphernl) [reply pillar.example]: Fixed.

**Comment by aboe76** [comment]
@psmiraglia are you satisfied ? then we can merge it?

**Comment by psmiraglia** [comment]
Hey @aboe76 and @syphernl, sorry for the delay by I was a little bit busy in the last days. The changes look good to me so we can merge!

**Comment by psmiraglia** [comment]
@aboe76 @syphernl I just realised that we made a mistake with the semantic versioning. We added Heartbeat support a.k.a. a new feature. According to semver rules, the new version should be `0.2.0` (backward compatible new feature) instead of `0.1.2` (backward compatible hotfix). If you agree, I can fix it.

**Comment by aboe76** [comment]
@psmiraglia If t's not too much to ask please do, we are in the process of adding semver to saltstack formulas, see the https://github.com/saltstack-formulas/template-formula for the progress

**Pull request merged**